sky/cli.py

```diff
@@ -3032,7 +3033,9 @@ def _output():
                         cpu_str = str(int(cpu_count))
                     else:
                         cpu_str = f'{cpu_count:.1f}'
-                mem_str = f'{item.memory:.0f}GB' if not pd.isna(
+                device_memory_str = (f'{item.device_memory:.0f}GB' if
+                                     not pd.isna(item.device_memory) else '-')
```

Comment:
What is the definition of device memory? IIRC, it is the amount of memory in a single device and does not depend on the device count, right? Then what about TPUs?

Reply:
Added some definition to `-h`. Wdyt? For TPU, we can defer adding documentation for it since we don't have that info in the catalog.